webroot/src/pages/Home/Home.ts

```diff
@@ -6,15 +6,50 @@
 //
 
 import { Component, Vue } from 'vue-facing-decorator';
-import Section from '@components/Section/Section.vue';
-import StateUpload from '@components/StateUpload/StateUpload.vue';
+import { Compact } from '@models/User/User.model';
 
 @Component({
     name: 'HomePage',
-    components: {
-        Section,
-        StateUpload,
-    }
+    components: {}
 })
 export default class Home extends Vue {
+    //
+    // Data
+    //
+    defaultCompact = Compact.ASLP;
```

Comment:
Same question here re: default compact. Also, could we easily use this defined default universally in the other file?

Reply:
I think this case is the broader fallback case. Here would end up populating the store with this default. The StateUpload case was a little more controversial. There was back and forth about whether that screen should have a compact selector or not (the original designs call for it, but I think user research has an item to get feedback). If you felt it best to remove the specific default from the StateUpload location, just let me know.

Reply:
Got it. Am I correct to say that, within `StateUpload`, the store's compact should _never_ be null since it's set in this component?

Or more likely, am I wrongly assuming `Home` is the parent component of all other pages? Is Home just a placeholder Home page for now? (It doesn't look like it has any contact at the moment.)

Reply:
Home is just another page in this case, not really a parent component of much. Just a landing page, kind of silent in this case, where we can direct the user to their appropriate starting location. No content for now.